patches/crosstool-NG/1.22.1/series

```diff
@@ -0,0 +1 @@
+Use-awk-hardlink.patch
```

Comment:
What problem does this patch solve?  The stg patch header should contain the use information about why the patch is needed and what it is solving.